libs/feature/map/src/lib/utils/map-utils.service.ts

```diff
@@ -169,4 +172,26 @@ export class MapUtilsService {
         })
     )
   }
+
+  prioritizePageScroll(interactions: Collection<Interaction>) {
+    interactions.clear()
+    interactions.extend(
+      defaults({ dragPan: false, mouseWheelZoom: false })
+        .extend([
+          new DragPan({
+            condition: dragPanCondition,
+          }),
+          new MouseWheelZoom({
+            condition: platformModifierKeyOnly,
+          }),
+        ])
+        .getArray()
+    )
+  }
+}
+
+export const dragPanCondition = (event) => {
+  return (
+    (this as DragPan).getPointerCount() === 2 || platformModifierKeyOnly(event)
```

Comment:
since this is an arrow function, `this` is undefined!

Reply:
This works:
```
export function dragPanCondition(this: DragPan, event) {
  return this.getPointerCount() === 2 || platformModifierKeyOnly(event)
}
```

Reply:
Thanks, indeed I missed that in the end.